### ISSUE TITLE
Vignette workflow for incremental updates

### ISSUE THREAD

**Issue by petersbrC** [jtlovell/GENESPACE#15]
Hello, this is a feature request.  

I was wondering if you could add a workflow showing how to best perform incremental updates for maximal existing results reuse when you want to add new genomes?

**Comment by jtlovell** [comment]
So, how can you re-run genespace just adding a new genome? 
If so, this is something planned, but not an easy feature to add. It will not be in the next major release ~June2022, but probably will be included in the following one ~Jan2023. Although I can't guarantee it. 
Sorry I can't be of more help.
John

**Comment by petersbrC** [comment]
Thanks for the response.  Its a very tough problem to do well and am looking forward to seeing your take on it.  Kudos on creating a bioinformatics system that works right away without a a ton of tweaking, I really like it.

I did have some issues specifically with the riparian plots which I thought might be linked to not having ImageMagick installed on my image.  They went away after installing it as I saw your environment had "magick" loaded in your session.  Not sure because I had also tried installing a number of orther missing libraries but after I added the OS dev and normal installation packages for ImageMagick the riparian plots were working.  Just FYI in case it represents another dependency for the install.
